boards/arm/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -226,3 +231,7 @@ zephyr_udc0: &usb1 {
 &gpt_hw_timer {
 	status = "okay";
 };
+
+&iomuxcgpr {
+	status = "okay";
+};
```

Comment:
note: if you make `CONFIG_PINCTRL=y` default, I think DT node should also be defaulted to `okay`